xcube/core/store/fs/impl/mldataset.py

```diff
@@ -225,10 +236,34 @@ def write_data(self,
         return data_id
 
 
-_MIN_CACHE_SIZE = 1024 * 1024  # 1 MiB
+class DatasetLevelsFsDataAccessor(MultiLevelDatasetLevelsFsDataAccessor):
+    """
+    Opener/writer extension name "dataset:levels:<protocol>".
+    """
```

Comment:
So, if I see this right, this accessor allows to (a) open datasets with multiple levels as if they only had one and (b) save a dataset with a single level as a multileveldataset with multiple levels. If this is so, I'd like it to be documented more. If it is not so, documentation is even more important.

Reply:
This is documented in the `plugin.py`. To view, please use CLI `xcube io opener list`.

Reply:
Note, it is not the intention of this PR to add docs or to enhance the framework. This is just about fixing code and testing issues.

Reply:
> This is documented in the `plugin.py`. To view, please use CLI `xcube io opener list`.

Okay. I'm not 100% happy with the phrasing, but this is clearly outside the scope of this PR.